src/main/java/blackjack/domain/bet/PlayerBets.java

```diff
@@ -0,0 +1,30 @@
+package blackjack.domain.bet;
+
+import blackjack.domain.player.Dealer;
+import blackjack.domain.player.Player;
+import blackjack.domain.result.GameResult;
+import blackjack.domain.result.PlayerProfits;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class PlayerBets {
+
+    private final Map<Player, BetAmout> playerBets;
+
+    public PlayerBets(Map<Player, BetAmout> playerBets) {
+        this.playerBets = playerBets;
+    }
+
+    public PlayerProfits calculateProfitResult(Dealer dealer) {
+        return new PlayerProfits(playerBets.keySet().stream()
+                .collect(Collectors.toMap(
+                        player -> player,
+                        player -> (calculatePlayerProfit(player, dealer)))));
+    }
```

Comment:
괄호 정리 추천! (람다식에서 사용하지 않아도 되는 괄호 제거)

```suggestion
    public PlayerProfits calculateProfitResult(Dealer dealer) {
        return new PlayerProfits(playerBets.keySet().stream()
                .collect(Collectors.toMap(
                        player -> player,
                        player -> calculatePlayerProfit(player, dealer)
                )));
    }
```